src/Nuget.PackageIndex.VisualStudio/ProjectKFilter.cs

```diff
@@ -43,8 +86,114 @@ public bool IsProjectSupported(string filePath)
                     // TODO Package manager console?
                     Debug.Write(string.Format("{0} \r\n {1}", e.Message, e.StackTrace));
```

Comment:
Just to confirm - we are ok for user to see actual exceptions and stack trace in the output? We just tried to avoid that in WTE since it would look weird...